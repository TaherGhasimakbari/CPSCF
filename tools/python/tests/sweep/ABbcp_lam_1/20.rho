 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.3110225019E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  3.000000002235E-01  6.999999997765E-01       0     1
  1.132355421877E-01 -1.132355421882E-01       1     2
  1.715926157701E-02 -1.715926158071E-02       2     2
  1.119421464814E-03 -1.119421465026E-03       3     2
 -1.352491063962E-04  1.352491064205E-04       4     2
 -5.017492836747E-05  5.017492843493E-05       5     2
 -6.490274389755E-06  6.490274497699E-06       6     2
 -1.111536362366E-07  1.111536706314E-07       7     2
  1.240473339474E-07 -1.240473565175E-07       8     2
  2.608163946180E-08 -2.608165205188E-08       9     2
  2.179018529047E-09 -2.179022623675E-09      10     2
 -1.746920980010E-10  1.746890469335E-10      11     2
 -8.355543699247E-11  8.355493898989E-11      12     2
 -1.213299539810E-11  1.213265481916E-11      13     2
 -4.065456795135E-13  4.060910604112E-13      14     2
  2.127530384249E-13 -2.125563647949E-13      15     2
  7.161632398223E-14 -7.122080702970E-14      16     1
