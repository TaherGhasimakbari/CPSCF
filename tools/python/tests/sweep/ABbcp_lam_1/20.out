 format  1  0

MONOMERS            
N_monomer           
                   2
kuhn                
    1.0000000000E+00    1.0000000000E+00

CHAINS              
N_chain             
                   1
N_block             
                   2
block_monomer       
                   1                   2
block_length        
    3.0000000022E-01    6.9999999978E-01

COMPOSITION         
ensemble            
                   0
phi_chain           
    1.0000000000E+00

INTERACTION         
interaction_type    
               'chi'
chi                 
    1.5000000011E+01

UNIT_CELL           
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.3110225019E+00

DISCRETIZATION      
ngrid               
                  32
chain_step          
    1.0000000000E-02

FILE_PREFIXES       
input_prefix        
               'in.'
output_prefix       
              'ref/'

BASIS               
group_name          
                '-1'

ITERATE             
max_itr             
                  20
error_max           
    1.0000000000E-08
domain              
                   T
itr_algo            
                'NR'
N_cut               
                  17

SWEEP               
s_max               
    2.0000000000E+01
d_block_length      
   -1.0000000000E-02    1.0000000000E-02
end_increments      

FINISH              

THERMO              
f_Helmholtz         
    2.1475504379E+00
f_homo              
    2.1500000037E+00
pressure            
    3.9532299701E+00
mu_chain            
    6.1007804080E+00
stress              
    1.8466130515E-11

STATISTICS          
N_star              
                  17
Final Error         
    1.8466130515E-09
Iterations          
                  19
Basis Time          
    2.6167106628E-01
SCF Time            
    1.4863705635E-01
